types/luxon/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6", "ES2020.Intl"
```

Comment:
is this correct?
https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/tsconfig.json#L553-L557
(formally, for devex, it does not pass validation in editors)

Reply:
@peterblazejewicz  It's present in ts for quite some time, but only ts4.2 version is causing problems. FWIW `es2020` itself gives the same results, so we can replace for that.

Reply:
yup, let's swap, thx!

Reply:
@peterblazejewicz fixed, let me know if there is something else